core/templates/pages/exploration-editor-page/editor-navigation/editor-navigation.directive.html

```diff
@@ -231,7 +231,7 @@
     </a>
   </li>
 
-  <li ng-if="isLargeScreen && isImprovementsTabEnabled()" ng-class="{'active': getActiveTabName() === 'improvements'}"
+  <li ng-if="isLargeScreen && isImprovementsTabEnabled" ng-class="{'active': getActiveTabName() === 'improvements'}"
```

Comment:
We have a convention in this codebase to only start with verbs when referring to methods. Perhaps screenIsLarge && improvementsTabIsEnabled.

Reply:
Done.